code/modules/power/smes.dm

```diff
@@ -39,7 +39,7 @@
 
 /obj/machinery/power/battery/smes/initialize()
 	..()
-	spawn(5)
+	spawn(50)
```

Comment:
I understand you're trying to hack around the map loader not being in a good startup location but don't hack around like this to fix it